integrasjonspunkt/pom.xml

```diff
@@ -75,12 +72,6 @@
                                 <version>1.5.18</version>
                                 <scope>runtime</scope>
                             </dependency>
-                            <dependency>
```

Comment:
Treng ein ikkje denne lenger?

Reply:
Nei, eneste vi brukte derfra var collapse for indeks/oversikt på venstre side, så bare estetikk. Kan vurdere å dra den inn igjen om den havner på maven central.